Point_set_processing_3/examples/Point_set_processing_3/CMakeLists.txt

```diff
@@ -148,6 +149,16 @@ else()
   message(STATUS "NOTICE: Some of the executables in this directory require Eigen 3.1 (or greater), and will not be compiled.")
 endif()
 
+find_path(CYCODEBASE_INCLUDE "cySampleElim.h" DOC "Path to the cyCodeBase directory (https://github.com/cemyuksel/cyCodeBase)")
+
+if(NOT CYCODEBASE_INCLUDE)
+  message( STATUS "poisson_eliminate will be compiled without the cyCodeBase library (https://github.com/cemyuksel/cyCodeBase)")
+else()
+  message(CYCODEBASE_INCLUDE="${CYCODEBASE_INCLUDE}")
+  target_compile_definitions( poisson_eliminate PRIVATE CGAL_USE_CY )
+  target_include_directories( poisson_eliminate AFTER PRIVATE ${CYCODEBASE_INCLUDE} )
+endif()
+
```

Comment:
Is that use of `<cySampleElim.h>` something we want to integrated in CGAL as it is?

Reply:
I removed the reference to the original implementation.